common/lib/xmodule/xmodule/js/src/video/08_video_speed_control.js

```diff
@@ -109,6 +110,13 @@ function (Iterator) {
             speedsContainer.html(speedsList.join(''));
             this.speedLinks = new Iterator(speedsContainer.find('.speed-option'));
             this.state.el.find('.secondary-controls').prepend(this.el);
+            if (this.state.speed) {
+                initialSpeed = this.state.speed;
+            } else {
+                // default if no speed is set
+                initialSpeed = "1.0";
```

Comment:
Yep. It looks like `render()` receives `speed` as a variable, and it's always set, so I'm using it instead of re-setting variables.